contracts/interchain-token-service/src/msg.rs

```diff
@@ -33,6 +34,13 @@ pub enum ExecuteMsg {
     /// The admin is allowed to remove the ITS address of a chain for emergencies.
     #[permission(Elevated)]
     DeregisterItsContract { chain: ChainNameRaw },
+
+    #[permission(Governance)]
+    SetChainConfig {
+        chain: ChainNameRaw,
+        max_uint: Uint256,
+        max_target_decimals: u8,
+    },
 }
 
 #[cw_serde]
```

Comment:
add a query to see the registered config per chain. If we combine the two configs as per above, we can simplify the query as well

Reply:
later